File: app/src/main/java/com/example/lucas/haushaltsmanager/Utils/WeekdayUtils.java
package com.example.lucas.haushaltsmanager.Utils;

import android.content.Context;

import com.example.lucas.haushaltsmanager.R;

public class WeekdayUtils {
    public static final int MONDAY = 0;

    private final String[] mWeekdays;

    public WeekdayUtils(Context context) {

        mWeekdays = context.getResources().getStringArray(R.array.weekdays);
    }

    public String getWeekday(int weekdayIndex) {
        return mWeekdays[weekdayIndex];
    }

    public int getWeekdayIndex(String weekday) {
        for (int i = 0; i < mWeekdays.length; i++) {
            if (mWeekdays[i].equals(weekday))
                return i;
        }

        return -1;
    }

    public String[] getWeekdays() {
        return mWeekdays;
    }
}

File: app/src/main/java/com/example/lucas/haushaltsmanager/Utils/WeekdayUtils.kt
package com.example.lucas.haushaltsmanager.Utils

import android.content.Context
import com.example.lucas.haushaltsmanager.R

class WeekdayUtils(var context: Context) {
    val weekdays: Array<String> = context.resources.getStringArray(R.array.weekdays)

    fun getWeekday(weekdayIndex: Int): String {
        return weekdays[weekdayIndex];
    }

    fun getWeekdayIndex(weekday: String): Int {
        return weekdays.indexOf(weekday);
    }
}